modules/installation-aws-permissions.adoc

```diff
@@ -9,287 +9,172 @@ When you attach the `AdministratorAccess` policy to the IAM user that you create
+* `elasticloadbalancing:AddTags`
+* `elasticloadbalancing:ApplySecurityGroupsToLoadBalancer`
+* `elasticloadbalancing:AttachLoadBalancerToSubnets`
+* `elasticloadbalancing:CreateListener`
+* `elasticloadbalancing:CreateLoadBalancer`
+* `elasticloadbalancing:CreateLoadBalancerListeners`
+* `elasticloadbalancing:CreateTargetGroup`
+* `elasticloadbalancing:ConfigureHealthCheck`
+* `elasticloadbalancing:DeleteLoadBalancer`
+* `elasticloadbalancing:DeregisterInstancesFromLoadBalancer`
+* `elasticloadbalancing:DeregisterTargets`
+* `elasticloadbalancing:DescribeInstanceHealth`
+* `elasticloadbalancing:DescribeListeners`
+* `elasticloadbalancing:DescribeLoadBalancers`
+* `elasticloadbalancing:DescribeLoadBalancerAttributes`
+* `elasticloadbalancing:DescribeTags`
+* `elasticloadbalancing:DescribeTargetGroupAttributes`
+* `elasticloadbalancing:DescribeTargetHealth`
+* `elasticloadbalancing:ModifyLoadBalancerAttributes`
+* `elasticloadbalancing:ModifyTargetGroup`
+* `elasticloadbalancing:ModifyTargetGroupAttributes`
+* `elasticloadbalancing:RegisterTargets`
+* `elasticloadbalancing:RegisterInstancesWithLoadBalancer`
+* `elasticloadbalancing:SetLoadBalancerPoliciesOfListener`
+
+.Required IAM permissions for installation
+* `iam:AddRoleToInstanceProfile`
+* `iam:CreateInstanceProfile`
+* `iam:CreateRole`
+* `iam:DeleteInstanceProfile`
+* `iam:DeleteRole`
+* `iam:DeleteRolePolicy`
+* `iam:GetInstanceProfile`
+* `iam:GetRole`
+* `iam:GetRolePolicy`
+* `iam:GetUser`
+* `iam:ListInstanceProfilesForRole`
+* `iam:ListRoles`
+* `iam:ListUsers`
+* `iam:PassRole`
+* `iam:PutRolePolicy`
+* `iam:RemoveRoleFromInstanceProfile`
+* `iam:SimulatePrincipalPolicy`
+* `iam:TagRole`
+
+.Required Route53 permissions for installation
+* `route53:ChangeResourceRecordSets`
+* `route53:ChangeTagsForResource`
+* `route53:GetChange`
+* `route53:GetHostedZone`
+* `route53:CreateHostedZone`
+* `route53:DeleteHostedZone`
+* `route53:ListHostedZones`
+* `route53:ListHostedZonesByName`
+* `route53:ListResourceRecordSets`
+* `route53:ListTagsForResource`
+* `route53:UpdateHostedZoneComment`
+
+.Required S3 permissions for installation
+* `s3:CreateBucket`
+* `s3:DeleteBucket`
+* `s3:GetAccelerateConfiguration`
+* `s3:GetBucketCors`
+* `s3:GetBucketLocation`
+* `s3:GetBucketLogging`
+* `s3:GetBucketObjectLockConfiguration`
+* `s3:GetBucketReplication`
+* `s3:GetBucketRequestPayment`
+* `s3:GetBucketTagging`
+* `s3:GetBucketVersioning`
+* `s3:GetBucketWebsite`
+* `s3:GetEncryptionConfiguration`
+* `s3:GetLifecycleConfiguration`
+* `s3:GetReplicationConfiguration`
+* `s3:ListBucket`
+* `s3:PutBucketAcl`
+* `s3:PutBucketTagging`
+* `s3:PutEncryptionConfiguration`
+
+.S3 permissions that cluster Operators require
+* `s3:PutObject`
+* `s3:PutObjectAcl`
+* `s3:PutObjectTagging`
+* `s3:GetObject`
+* `s3:GetObjectAcl`
+* `s3:GetObjectTagging`
+* `s3:GetObjectVersion`
+* `s3:DeleteObject`
+
+.All permissions that are required to uninstall a cluster
```

Comment:
This should be "All *additional* permissions..." or some such.  You need some of the above permissions as well, and we don't currently distinguish between install-time and destroy-time permission requirements in the installer code-base (although we probably should).